mail/api.py

```diff
@@ -173,13 +215,14 @@ def send_financial_aid_email(cls, acting_user, financial_aid, subject, body):
             financial_aid (FinancialAid): the FinancialAid object this pertains to (recipient is pulled from here)
             subject (str): email subject
             body (str): email body
+            raise_for_status (bool): If true and we received a non 2xx status code from Mailgun, raise an exception
         Returns:
             requests.Response: response from Mailgun
         """
         from_address = cls.default_params()['from']
         to_address = financial_aid.user.email
-        response = cls.send_individual_email(subject, body, to_address)
-        if response.status_code == HTTP_200_OK:
+        response = cls.send_individual_email(subject, body, to_address, raise_for_status=raise_for_status)
+        if response.status_code == status.HTTP_200_OK:
```

Comment:
OK, that's true. But you should still check [`response.ok`](http://docs.python-requests.org/en/master/api/#requests.Response.ok) instead of comparing the status code with 200. There are other successful status codes beyond 200.